<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-11-11T17:35:14Z" name="fi.tuni.prog3.junitorder.newJavaFileNGTest" finished-at="2022-11-11T17:35:14Z" duration-ms="17">
    <groups>
    </groups>
    <test started-at="2022-11-11T17:35:14Z" name="Command line test" finished-at="2022-11-11T17:35:14Z" duration-ms="17">
      <class name="fi.tuni.prog3.junitorder.newJavaFileNGTest">
        <test-method is-config="true" signature="setUpClass()[pri:0, instance:fi.tuni.prog3.junitorder.newJavaFileNGTest@670b40af]" started-at="2022-11-11T19:35:14Z" name="setUpClass" finished-at="2022-11-11T19:35:14Z" duration-ms="6" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method is-config="true" signature="setUpMethod()[pri:0, instance:fi.tuni.prog3.junitorder.newJavaFileNGTest@670b40af]" started-at="2022-11-11T19:35:14Z" name="setUpMethod" finished-at="2022-11-11T19:35:14Z" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method signature="testMain()[pri:0, instance:fi.tuni.prog3.junitorder.newJavaFileNGTest@670b40af]" started-at="2022-11-11T19:35:14Z" name="testMain" finished-at="2022-11-11T19:35:14Z" duration-ms="2" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The test case is a prototype.]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The test case is a prototype.
	at org.testng.Assert.fail(Assert.java:94)
	at fi.tuni.prog3.junitorder.newJavaFileNGTest.testMain(newJavaFileNGTest.java:48)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:577)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testMain -->
        <test-method is-config="true" signature="tearDownMethod()[pri:0, instance:fi.tuni.prog3.junitorder.newJavaFileNGTest@670b40af]" started-at="2022-11-11T19:35:14Z" name="tearDownMethod" finished-at="2022-11-11T19:35:14Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:fi.tuni.prog3.junitorder.newJavaFileNGTest@670b40af]" started-at="2022-11-11T19:35:14Z" name="tearDownClass" finished-at="2022-11-11T19:35:14Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- fi.tuni.prog3.junitorder.newJavaFileNGTest -->
    </test> <!-- Command line test -->
  </suite> <!-- fi.tuni.prog3.junitorder.newJavaFileNGTest -->
</testng-results>
